tests/saitests/py3/sai_qos_tests.py

```diff
@@ -749,18 +749,18 @@ def runTest(self):
                 print(list(map(operator.sub, pg_cntrs, pg_cntrs_base)),
                       file=sys.stderr)
                 for i in range(0, PG_NUM):
+                    # LACP packets are mapped to queue0 and tcp syn packets for BGP to queue4
+                    # CPU sends LACP/LLDP/BGP control pkts to queue7
+                    # So for those queues the count could be more
```

Comment:
all control traffic is always mapped to q7. on which platform do you see lacp and bgp packets mapped to q0 and q4 respectively?

Reply:
> all control traffic is always mapped to q7. on which platform do you see lacp and bgp packets mapped to q0 and q4 respectively?

The LLDP and lacp packets received from peer will go to Q0 since it is untagged non ip packet. The BGP packets from peer are going to Q4 with the TOS value 0xC0.

Reply:
@neethajohn , this was behavior on DNX. Recent SAI change fixed this issue.

Reply:
Checked with @neethajohn. We need to check why the packets aren't coming on PG=7. all control packets should go to PG=7.

Reply:
@neethajohn if you are refering to this PR https://github.com/sonic-net/sonic-mgmt/pull/8097 and related fixes -- isn't not for the TX packets only ? i.e all the control packets being tx'ed should use queue 7.

Here @saksarav-nokia is taking about the queue used for packets RX, coming into the front panel ports. Is that also always queue 7 irrespective of the DSCP or other fields in the packet?